src/Plugin/MediaEmbedDialog/Media.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Drupal\stanford_media\Plugin\MediaEmbedDialog;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\stanford_media\Plugin\MediaEmbedDialogBase;
+
+/**
+ * Modifies all embed dialogs for all media types.
+ *
+ * @MediaEmbedDialog(
+ *   id = "all_media"
+ * )
+ */
+class Media extends MediaEmbedDialogBase {
+
+  /**
+   * {@inheritDoc}
+   */
+  public function isApplicable() {
+    return TRUE;
+  }
```

Comment:
![](https://media0.giphy.com/media/3orif2teAM5HwdVMB2/source.gif)

Reply:
Why have this function if all it does is return true?

Reply:
https://github.com/SU-SWS/stanford_media/blob/drupal-8.8-upgrade/src/Plugin/MediaEmbedDialog/File.php#L21
other plugins return false conditionally